gdnative/src/variant_array.rs

```diff
@@ -118,6 +142,27 @@ impl VariantArray {
             (get_api().godot_array_hash)(&self.0)
         }
     }
+
+    pub fn sort(&mut self) {
+        unsafe {
+            (get_api().godot_array_sort)(&mut self.0)
+        }
+    }
+
+    // TODO
+    // pub fn sort_custom(&mut self, obj: ?, s: ?) {
+    //     unimplemented!()
+    // }
+
+    // pub fn bsearch(&mut self, val: (), before: bool) -> i32 {
```

Comment:
@nical  Could you give me some tips on how to implement these bindings:
`sort_custom` - `godot_array_sort_custom`, `bsearch` - `godot_array_bsearch` and `bsearch_custom` - `godot_array_bsearch_custom`.

Reply:
Those look tricky. I'll have a look when I have some time, but it's fine to leave them out for now.